org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/tree/AbstractTreeNode.java

```diff
@@ -1106,66 +996,29 @@ public Object validateValue(ICell cell, Object value) {
     return value;
   }
 
-  /**
-   * The extension delegating to the local methods. This Extension is always at the end of the chain and will not call
-   * any further chain elements.
-   */
-  protected static class LocalTreeNodeExtension<OWNER extends AbstractTreeNode> extends AbstractExtension<OWNER> implements ITreeNodeExtension<OWNER> {
-
-    public LocalTreeNodeExtension(OWNER owner) {
-      super(owner);
-    }
-
-    @Override
-    public void execDecorateCell(TreeNodeDecorateCellChain chain, Cell cell) {
```

Comment:
This three extension methods are currently not available on a PageExtension. I thing they should be pushed down.